### DISCUSSION TITLE
FileGlobLivestream opens a stream for each file

### DISCUSSION THREAD

**Discussion by Mikulskii** [scivision/PyLivestream#104] [Ideas]
Hello! The script works fine, but there is a problem for me:
When playing a list of files via the _FileGloblivestream /playlist twitch -glob "*.mp4" -loop_ command, then when starts playing the next video file from the list, the broadcast restarts. 
Is there any way around this? 

Ubuntu 20.04, ffmpeg 4, Python 3.9

**Comment by Ch41r05** [comment]
Hi @Mikulskii , did you solve the issue you reported? If yes, how did you go about it? I'm encountering the same problem, given that looking at the python code I understand the stream is reset for each file.

**Comment by Mikulskii** [comment]
@Ch41r05 Hi! I closed this issue, because among the solved ones I found that this script is not focused on playlists.
As a result, I use the ffplayout script: https://github.com/ffplayout

**Comment by Ch41r05** [comment]
@Mikulskii thanks for the prompt response! Would you mind sharing what solution you implemented? I made my own fork where I fixed this issue by using a playlist generated at runtime from all files, eventually filtered by glob, present in the given folder. What doesn't work with this approach is randomization and captions.

**Comment by Mikulskii** [comment]
@Ch41r05 I'm not a programmer, just a user. 
n general, I use ffplayout-script: https://github.com/ffplayout  to randomly play videos from a folder. Check it out. Maybe you will find something useful from the code.
I wrote a post on my blog about how I set up everything and use it: https://mikulski.rocks/how-i-made-my-stream-radio-24-7/
